File: java-samples/persistence-carinsurance/contracts/src/main/java/com/r3/developers/samples/persistence/states/Claim.java
package com.r3.developers.samples.persistence.states;

import net.corda.v5.base.annotations.CordaSerializable;

/**
 * Simple POJO class for the claim details.
 */
@CordaSerializable
public class Claim {

    private final String claimNumber;
    private final String claimDescription;
    private final int claimAmount;

    public Claim(String claimNumber, String claimDescription, int claimAmount) {
        this.claimNumber = claimNumber;
        this.claimDescription = claimDescription;
        this.claimAmount = claimAmount;
    }

    public String getClaimNumber() {
        return claimNumber;
    }

    public String getClaimDescription() {
        return claimDescription;
    }

    public int getClaimAmount() {
        return claimAmount;
    }
}

File: kotlin-samples/persistence-carinsurance/contracts/src/main/kotlin/com/r3/developers/samples/persistence/states/Claim.kt
package com.r3.developers.samples.persistence.states

import net.corda.v5.base.annotations.CordaSerializable

/**
 * Simple POJO class for the claim details.
 */
@CordaSerializable
data class Claim(val claimNumber: String,
                 val claimDescription: String,
                 val claimAmount: Int)